support-frontend/assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.jsx

```diff
@@ -80,6 +82,7 @@ const mapDispatchToProps = (dispatch: Function) => ({
     (paymentRequest: Object) => { dispatch(setStripePaymentRequestObject(paymentRequest)); },
   updateEmail: (email: string) => { dispatch(updateEmail(email)); },
   setStripePaymentRequestButtonClicked: () => { dispatch(setStripePaymentRequestButtonClicked()); },
+  setAssociatedPaymentMethod: () => { dispatch(updatePaymentMethod(Stripe)); },
```

Comment:
I think I prefer this being a separate method than an additional `dispatch(updatePaymentMethod(Stripe))` line being put inside `setStripePaymentRequestButtonClicked`, what do you think @tomrf1 ?